### PR TITLE
Updated source copyright header

### LINKED ISSUE #992: Update source code copyrights header
I am going to sync source code copyright header with the project + update copyright in-app scroller to match.

This means:
* I will shorten the copyright header
* I will remove list of contributors from source header
* I will sync in-app scroller content with [credits page](https://github.com/logisim-evolution/logisim-evolution/blob/develop/docs/credits.md)

Please:
* check if [credits page](https://github.com/logisim-evolution/logisim-evolution/blob/develop/docs/credits.md) needs updating
* other comments?

### PR DESCRIPTION
**Pull request by MarcinOrlowski**
No other changes but mass search/replace + a few added headers extra.

Because every single file was touched, Checkstyle will run on all of them too, failing miserably. This is expected and should be ignored.

Partially fixes #992

### CODE REVIEW

**Comment by MarcinOrlowski** [comment]
No other changes but mass search/replace + a few added headers extra.

File: build.gradle.kts [hunk 1]
```diff
@@ -1,29 +1,10 @@
 /*
- * This file is part of logisim-evolution.
+ * Logisim-evolution - digital logic design tool and simulator
+ * Copyright by Logisim-evolution developers
```
Comment (BFH-ktt1) [inline build.gradle.kts:3]: ```Copyright by Logisim-evolution developers``` I would change in ```Copyright by the Logisim-evolution developers```

**Review by BFH-ktt1 — CHANGES_REQUESTED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (MarcinOrlowski) [reply build.gradle.kts:3]: Replaced.

**Comment by MarcinOrlowski** [comment]
BTW: I recommend SQUASH merge.

File: src/main/java/com/cburch/contracts/BaseWindowFocusListenerContract.java [hunk 2]
```diff
@@ -1,3 +1,12 @@
+/*
+ * Logisim-evolution - digital logic design tool and simulator
+ * Copyright by Logisim-evolution developers
```
Comment (BFH-ktt1) [inline src/main/java/com/cburch/contracts/BaseWindowFocusListenerContract.java:3]: Not all have been replaced, multiple places

**Review by BFH-ktt1 — CHANGES_REQUESTED** [review]

File: src/main/java/com/cburch/contracts/BaseWindowFocusListenerContract.java (hunk 2 above)
Comment (MarcinOrlowski) [reply src/main/java/com/cburch/contracts/BaseWindowFocusListenerContract.java]: Last 11 files. Fixed

**Review by BFH-ktt1 — APPROVED** [review]

**Pull request merged**